plugins/woocommerce/tests/e2e-pw/tests/merchant/create-post.spec.js

```diff
@@ -0,0 +1,59 @@
+const { test, expect, request } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
+
+const postTitle = `Post-${ new Date().getTime().toString() }`;
+
+test.describe( 'Can create a new post', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.afterAll( async ( { baseURL } ) => {
+		const base64auth = btoa( `${ admin.username }:${ admin.password }` );
+		const wpApi = await request.newContext( {
+			baseURL: `${ baseURL }/wp-json/wp/v2/`,
+			extraHTTPHeaders: {
+				Authorization: `Basic ${ base64auth }`,
+			},
+		} );
+		let response = await wpApi.get( `posts` );
+		const allPosts = await response.json();
+
+		await allPosts.forEach( async ( post ) => {
+			if ( post.title.rendered === postTitle ) {
+				response = await wpApi.delete( `posts/${ post.id }` );
+				expect( response.ok() ).toBeTruthy();
+			}
+		} );
+	} );
+
+	test( 'can create new post', async ( { page } ) => {
+		await page.goto( 'wp-admin/post-new.php', {
+			waitUntil: 'networkidle',
+		} );
+
+		await page
+			.getByRole( 'textbox', { name: 'Add Title' } )
+			.fill( postTitle );
+
+		await page.getByRole( 'button', { name: 'Add default block' } ).click();
+
+		await page
+			.getByRole( 'document', {
+				name:
+					'Empty block; start writing or type forward slash to choose a block',
+			} )
+			.fill( 'Test Post' );
+
+		await page
+			.getByRole( 'button', { name: 'Publish', exact: true } )
```

Comment:
I haven't tried it without, but why did you choose to use `exact: true` here?  Looking at it, I'd think that that a Role selector + Publish text would be specific enough, or were you getting multiple elements returned?

Reply:
Looks like multiple elements on this one:

`Error: strict mode violation: getByRole('button', { name: 'Publish' }) resolved to 2 elements:`

```
<button type="button" aria-disabled="false" aria-expande…>Publish</button> aka getByRole('button', { name: 'Publish', exact: true })
    2) <button type="button" aria-expanded="false" class="compo…>Open publish panel</button> aka getByRole('button', { name: 'Open publish panel' })
```

Reply:
Yes, there were two elements matching `Publish`, so I used `exact`